advanced_filters/locale/it/LC_MESSAGES/django.po

```diff
@@ -0,0 +1,146 @@
+# SOME DESCRIPTIVE TITLE.
+# Copyright (C) YEAR THE PACKAGE'S COPYRIGHT HOLDER
+# This file is distributed under the same license as the PACKAGE package.
+# FIRST AUTHOR <EMAIL@ADDRESS>, YEAR.
+#
+#, fuzzy
+msgid ""
+msgstr ""
+"Project-Id-Version: PACKAGE VERSION\n"
+"Report-Msgid-Bugs-To: \n"
+"POT-Creation-Date: 2018-01-25 09:49+0100\n"
+"PO-Revision-Date: YEAR-MO-DA HO:MI+ZONE\n"
+"Last-Translator: FULL NAME <EMAIL@ADDRESS>\n"
+"Language-Team: LANGUAGE <LL@li.org>\n"
+"Language: \n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Plural-Forms: nplurals=4; plural=(n%10==1 && n%100!=11 ? 0 : n%10>=2 && n"
+"%10<=4 && (n%100<12 || n%100>14) ? 1 : n%10==0 || (n%10>=5 && n%10<=9) || (n"
+"%100>=11 && n%100<=14)? 2 : 3);\n"
+
+#: admin.py:24
+msgid "Advanced filters"
+msgstr "Filtri avanzati"
+
+#: admin.py:81
+msgid "Advanced filter added successfully."
+msgstr "Filtro aggiunto correttamente."
+
+#: forms.py:58
+msgid "Equals"
+msgstr "Uguale"
+
+#: forms.py:59
+msgid "Contains"
+msgstr "Contiene"
+
+#: forms.py:60
+msgid "One of"
+msgstr "Uno di"
+
+#: forms.py:61
+msgid "DateTime Range"
+msgstr "Intervallo data e ora"
+
+#: forms.py:62
+msgid "Is NULL"
+msgstr "è Nullo"
+
+#: forms.py:63
+msgid "Is TRUE"
+msgstr "è Vero"
+
+#: forms.py:64
+msgid "Is FALSE"
+msgstr "è Falso"
+
+#: forms.py:65
+msgid "Less Than"
+msgstr "Minore di"
+
+#: forms.py:66
+msgid "Greater Than"
+msgstr "Maggiore di"
+
+#: forms.py:67
+msgid "Less Than or Equal To"
+msgstr "Minore o uguale di"
```

Comment:
Ciao @fcorallini !

What do you think of `Minore o uguale a` for this string?
Same for line 73.

Reply:
Looks good to me!

Reply:
@fcorallini are you going to address the issue ^?